src/main/java/com/Acrobot/ChestShop/Listeners/Modules/PriceRestrictionModule.java

```diff
@@ -73,12 +92,16 @@ private void load() {
                     configuration.save(file);
                     ChestShop.getBukkitLogger().log(Level.INFO, "Conversion finished!");
                 } catch (IOException e) {
-                    ChestShop.getBukkitLogger().log(Level.SEVERE, "Error while converting priceLimits.yml", e);
+                    ChestShop.getBukkitLogger().log(Level.SEVERE,
+                            "Error while converting priceLimits.yml", e);
                 }
             } else {
-                ChestShop.getBukkitLogger().log(Level.WARNING, "Could not convert numeric IDs in priceLimits.yml to Material names!");
-                ChestShop.getBukkitLogger().log(Level.WARNING, "If you want to automatically convert them you have to run this version on a pre 1.13 server.");
-                ChestShop.getBukkitLogger().log(Level.WARNING, "If you want to manually convert it and hide this message set the uses_materials key to true.");
+                ChestShop.getBukkitLogger().log(Level.WARNING,
+                        "Could not convert numeric IDs in priceLimits.yml to Material names!");
+                ChestShop.getBukkitLogger().log(Level.WARNING,
+                        "If you want to automatically convert them you have to run this version on a pre 1.13 server.");
+                ChestShop.getBukkitLogger().log(Level.WARNING,
+                        "If you want to manually convert it and hide this message set the uses_materials key to true.");
```

Comment:
Please keep the logs as they were before. I assume they conflicted with your IDE's max-line-length but that 
change makes it hard to see what actually changed in this commit so shouldn't be part of it.

Reply:
Changed - will let you mark as resolved on evaluation